pom.xml

```diff
@@ -44,6 +44,18 @@
             <organization>incub8 Software Labs</organization>
             <organizationUrl>http://www.incub8.de</organizationUrl>
         </developer>
+        <developer>
+            <name>Sven Haberer</name>
```

Comment:
I know it's not obvious from the existing entries, but I'd like to sort these "mechanically" based on the entire `<name>`, not "smart" on the last name, which may not even work for some people (with no or multiple last names).